docs/en/integrations/airbyte-and-clickhouse.md

```diff
@@ -35,25 +35,21 @@ description: Stream data into ClickHouse using Airbyte data pipelines
 
 In this section, we will display how to add a ClickHouse instance as a destination.
 
-1. Start your ClickHouse server (Airbyte is compatible with ClickHouse version `21.8.10.19` or above):
+1. Start your ClickHouse server (Airbyte is compatible with ClickHouse version `21.8.10.19` or above) or login to your ClickHouse cloud account:
```

Comment:
sorry, missed that one.  I would rather not have the "start clickhouse" step up top since if they are using Cloud they do not have to start it, they just need the details.